src/EnergyPlus/Psychrometrics.hh

```diff
@@ -356,45 +356,41 @@ namespace Psychrometrics {
 
         // Static locals
         static Real64 dwSave(-100.0);
-        static Real64 Tsave(-100.0);
         static Real64 cpaSave(-100.0);
 
         // check if last call had the same input and if it did just use the saved output
-        if ((Tsave == T) && (dwSave == dw)) return cpaSave;
+        if (dwSave == dw) return cpaSave;
 
         // compute heat capacity of air
         Real64 const w(max(dw, 1.0e-5));
-        Real64 const cpa((PsyHFnTdbW(T + 0.1, w) - PsyHFnTdbW(T, w)) * 10.0); // result => heat capacity of air {J/kg-C}
+        Real64 const cpa((1.00484e3 + w * 1.85895e3)); // result => heat capacity of moist air {J/kg-C}
```

Comment:
Is there a reference for this change?  The PR description is empty, and with lots of diffs to work through, it will be important to document where this change came from.